src/library.js

```diff
@@ -344,6 +344,23 @@ const createPilingJs = (rootElement, initOptions = {}) => {
   const activePile = new PIXI.Container();
   const normalPiles = new PIXI.Container();
 
+  const addToActivePile = pileGfx => {
```

Comment:
```suggestion
  const moveToActivePileLayer = pileGfx => {
```

I think this is a but clearer